data.sql

```diff
@@ -1,8 +1,15 @@
 /* Populate database with sample data. */
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES 
-('Agumon', '2020-02-03', 0, true, 10.23),
-('Gabumon', '2018-11-15', 2, true, 8),
-('Pikachu', '2021-01-07', 1, false, 15.04),
-('Devimon', '2017-05-12', 5, true, 11);
+VALUES
+	('Agumon', '2020-02-03', 0, true, 10.23),
+	('Gabumon', '2018-11-15', 2, true, 8),
+	('Pikachu', '2021-01-07', 1, false, 15.04),
+	('Devimon', '2017-05-12', 5, true, 11),
+  ('Charmander', '2020-02-08', 0, false, 11),
+  ('Plantmon', '2021-11-15', 2, true, 5.7),
+  ('Squirtle', '1993-04-02', 3, false, 12.13),
+  ('Angemon', '2005-06-12', 1, true, 45),
+  ('Boarmon', '2005-06-07', 7, true, 20.4),
+  ('Blossom', '1998-10-13', 3, true, 17),
+  ('Ditto', '2022-05-14', 4, true, 22);
```

Comment:
- [ ] Please from the project requirement, You should Insert the following data:
  - Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
  - Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
  - Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
  - Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
  - Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
  - Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.

The data to be inserted is 7 rows but you are inserting more than 7 and besides you are inserting wrong data as compared to information about the data to be inserted above :blush: For instance _Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg_ the data for the `weight _kg` column should be -11.0 but you inserted it as 11.0 :smile:

Reply:
Hi @Owusu-Desmond,

These first 4 animals are comming from the previous exercise:

`('Agumon', '2020-02-03', 0, true, 10.23),
	('Gabumon', '2018-11-15', 2, true, 8),
	('Pikachu', '2021-01-07', 1, false, 15.04),
	('Devimon', '2017-05-12', 5, true, 11),`

Do you want me to remove those from the `data.sql` file?